### ISSUE TITLE
Registration crashes with "Oops, an error occurred!"

### ISSUE THREAD

**Issue by kludikovsky** [derhansen/sf_event_mgt#306]
### Extension version
1.4.0

### Short problem description
Event Registraion crash

### Steps to reproduce
1.Select an event
2.Enter the data
3.Submit

### Expected behaviour
Register person to event (and send mails)

### Actual behaviour
Response:
"Oops, an error occurred! Code: 2016032409563731b96df6"

I found the following entry in the log 
[sf_event_error_log.txt](https://github.com/derhansen/sf_event_mgt/files/187499/sf_event_error_log.txt) (log Entry strucutured and 'beautified')

This is the TS 
![event ts](https://cloud.githubusercontent.com/assets/6077916/14014052/66a7949c-f1b0-11e5-8e4b-d7801a912f4d.jpg)

The error can be reproduced.
.

Basically I have just upgraded from 1.3.1-dev to 1.4.0 some days ago (maybe did no check then.
Update realurl. (so this might have had an influence as well, but not sure) 

### Server configuration
**T3 7.6.4:**

**PHP 5.6.15:**

**Comment by derhansen** [comment]
I can see in your log, that no registration object is passed to the saveRegistration action. Please check, if the registration works with the included templates. It is important, that the registration form has the registration object as shown here https://github.com/derhansen/sf_event_mgt/blob/master/Resources/Private/Templates/Event/Registration.html#L80

**Comment by kludikovsky** [comment]
It's in there.

I replaced my (modified 1.3.1) registration tempalte now with the included one an get the same error.

**Comment by kludikovsky** [comment]
I am just struggling with the difference between

`plugin.tx_sfeventmgt.view.templateRootPaths `

which is mostly mentioned in the documentation
`
and

`plugin.tx_sfeventmgt.templateRootPaths `

which is set in [the setup](https://github.com/derhansen/sf_event_mgt/blob/master/Configuration/TypoScript/setup.txt#L6)

as I have both in the TS.


Also the difference between the singular an plural notation

**Comment by derhansen** [comment]
Please checkout the lastest development branch. I think the bugfix from yesterday may help. If not, please drop me a message.

**Comment by kludikovsky** [comment]
sent you a PN with some data.

**Comment by kludikovsky** [comment]
Thanks for the feedback.

Just for completeness:
There seems to be a real issue when updating real_url and bootstrap_package.

I'll close this issue.

Tnx.

**Comment by kludikovsky** [comment]
Just to let you know for completeness

I have the statement 
`config.absRefPrefix = http://www.mydom.tld/`
in my tempalte.

When I comment this statement out registration works again.  
I might have had some inconsistencies with `www.mydom.tld` and`mydom..tld`.  
I am still investigating.
